ui/app/src/state/epics/pathNavigator.ts

```diff
@@ -375,15 +431,15 @@ export default [
             navigator.currentPath === parentPathOfTargetPath ||
             navigator.currentPath === parentOfTargetWithIndex
           ) {
-            actions.push(pathNavigatorBackgroundRefresh({ id: navigator.id }));
+            idsToRefresh.push(navigator.id);
           } /* else if (
             // Case (c) - Content epics load any item that's on the state already
             navigator.currentPath === getParentPath(parentPathOfTargetPath)
           ) {
             actions.push(fetchSandboxItem({ path: parentPathOfTargetPath }));
           } */
         });
-        return actions;
+        return of(pathNavigatorBulkBackgroundRefresh({ ids: idsToRefresh }));
```

Comment:
If we're returning a single action, you can switch from `mergeMap` to `map` and dismiss the use of `of`.
Do not dispatch if ids is empty